src/core/HorizontalTabs/HorizontalTabs.tsx

```diff
@@ -94,54 +103,49 @@ export const HorizontalTabs = (props: HorizontalTabsProps) => {
     setCurrentIndex(index);
   };
 
-  const mainClass = React.useCallback(() => {
-    switch (type) {
-      case 'borderless':
-        return 'iui-tabs-borderless';
-      case 'pill':
-        return 'iui-tabs-pill';
-      default:
-        return 'iui-tabs-horizontal';
-    }
-  }, [type]);
-
   return (
-    <>
+    <div className='iui-tabs-wrapper'>
       <ul
         className={cx(
-          mainClass(),
-          { 'iui-green': color === 'green' },
+          'iui-tabs',
+          {
+            [`iui-${type}`]: !!type,
```

Comment:
`type` is always present as we have default value for it.

Reply:
Fixed